app/main.py

```
@@ -24,9 +26,30 @@ async def lifespan(_app: FastAPI):
+            interval_seconds=settings.sweeper_interval_seconds,
+            max_requeue_attempts=settings.sweeper_max_requeue_attempts,
+        )
+        sweeper_task = asyncio.create_task(sweeper.run_forever(AsyncSessionLocal))
+        logger.info("Sweeper enabled")
+
     yield
     # Shutdown
     logger.info("Shutting down")
+    if sweeper_task is not None:
+        sweeper_task.cancel()
+        done, pending = await asyncio.wait({sweeper_task}, timeout=5)
```

Ruff (0.14.10)

Unpacked variable `done` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)